packages/app-todo-list/interbit.prod.config.js

```diff
@@ -0,0 +1,82 @@
+const path = require('path')
+const chainAliases = require('./src/constants/chainAliases')
+
+const PUBLIC_KEY =
```

Comment:
We've upgraded to 0.13.0 so we are no longer using armored pubkeys. These will need to be updated